Model/Paypal_Payment_Method.php

```diff
@@ -61,16 +63,19 @@ public function getConfig(): Config
     }
 
     /**
-     * Get the current transaction to be processed
-     *
-     * @var double $dAmount
-     * @var Order $oOrder
-     *
-     * @return \Wirecard\PaymentSdk\Transaction\Transaction
+     * @inheritdoc
      */
-    public function getTransaction(): Transaction
+    public function getTransaction(): \Wirecard\PaymentSdk\Transaction\Transaction
```

Comment:
Could this not simply be `public function getTransaction(): Transaction` ?

Reply:
👍